src/core/Akka.Coordination.Tests/TestLease.cs

```diff
@@ -7,20 +7,56 @@
 
 using System;
 using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using Akka.Actor;
 using Akka.Configuration;
-using Akka.Coordination;
 using Akka.Event;
 using Akka.TestKit;
 using Akka.TestKit.Xunit2;
 using Akka.Util;
 
-namespace Akka.Cluster.Tools.Tests
+namespace Akka.Coordination.Tests
```

Comment:
Most of the changes in this PR are due to moving this file around, so this PR is actually simpler than it appears.